hangman_lunch2.rb

```diff
@@ -0,0 +1,206 @@
+# Hangman
+require "colorize"
+
+class Hangman
+
+  def initialize
+    @answer = populate_answer
+    # Displayed line to fill in correct guesses
+    @answer_line = answer_line
+    # count of wrong answers
+    @wrong_count = 0
+    @letter_array = ["A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M", "N", "O", "P", "Q", "R", "S", "T", "U", "V", "W", "X", "Y", "Z"]
+    @letters_guessed = []
+
+    start
+    draw_board
+    play
+  end
+
+  def start
+    puts "Welcome to Star Trek Hangman! Good luck!"
+    # Keeps track of game play
+    @now_playing = true
+  end
+
+# ------------------
+# Create answer
+  def populate_answer
+    answer_array = ["PICARD", "GUINAN", "WORF", "WARP", "ENGAGE", "ENTERPRISE", "DATA", "ROMULAN", "VULCAN", "BETAZOID", "KLINGON", "FEDERATION", "REPLICATOR", "DARMOK", "HOLODECK", "PHASER", "TRICORDER", "BEAM", "TRANSPORTER"]
+
+    answer = answer_array.shuffle.pop
```

Comment:
For an easier way, arrays have a built in method to get a random element called `#sample`.

You could also shortcut this by doing:

   `return answer_array.shuffle.pop`